packages/logger/index.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+function __export(m) {
+    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
+}
+Object.defineProperty(exports, "__esModule", { value: true });
+// NOTE(bajtos) This file is used by VSCode/TypeScriptServer at dev time only
+__export(require("./src"));
```

Comment:
Please use the same contents of `index.js` as other packages are using. We need to switch between `lib` and `lib6` depending on the version of Node.js that we are running. See e.g. here: [packages/core/index.js](https://github.com/strongloop/loopback-next/blob/master/packages/core/index.js). Feel free to add missing `"use strict";` header though.